.vscode/settings.json

```diff
@@ -31,5 +31,8 @@
         "typechk",
         "uvicorn",
         "venv"
-    ]
+    ],
+    "yaml.schemas": {
```

Comment:
what was the need for adding this settings?

Reply:
It was throwing an error for the build.yml workflow file as vscode was unable to identify if the format to consider is either of "GitHub Workflow" format or "Hammerkit YAML Schema" format. To avoid that confusion, the setting was added.

reference: https://github.com/redhat-developer/vscode-yaml/issues/565#issuecomment-899097769